.github/workflows/build-ci.yml

```diff
@@ -70,8 +73,10 @@ jobs:
                     restore-keys: "${{ matrix.os }}-composer-"
 
             -   name: "Install dependencies"
-                run: composer update --no-interaction $([[ "${{ matrix.mode }}" == low-deps ]] && echo ' --prefer-lowest --prefer-stable')
-
+                run: |
+                  composer update --no-interaction \
+                  $([[ "${{ matrix.mode }}" == low-deps ]] && echo ' --prefer-lowest --prefer-stable') \
+                  $([[ "${{ matrix.mode }}" != dev ]] && echo ' --prefer-stable')
```

Comment:
Nit: I believe this may result in a duplicate `prefer-stable` option when the mode is `low-deps`, as the first and second conditions will match. It should be sufficient to use `--prefer-lowest` in the first check, and add `--prefer-stable` if the second condition matches.

Reply:
Fixed.